azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -1043,6 +1057,17 @@ func expandKubernetesClusterAddonProfiles(d *schema.ResourceData) map[string]*co
 		}
 	}
 
+	azurePolicy := profile["azure_policy"].([]interface{})
+	if len(azurePolicy) > 0 {
+		value := azurePolicy[0].(map[string]interface{})
+		enabled := value["enabled"].(bool)
+
+		addonProfiles["azurepolicy"] = &containerservice.ManagedClusterAddonProfile{
+			Enabled: utils.Bool(enabled),
+			Config:  nil,
```

Comment:
Is Config something we'll want to implement in the future? If so, can you add something like
```suggestion
                         // TODO add support for AddonProfile Config
			Config:  nil,
```

Reply:
I don't think we will - I think this is always just going to be a true / false flag.